optaweb-tsp-ui/package.json

```diff
@@ -2,43 +2,43 @@
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
-    "gh-pages": "^1.2.0",
+    "@patternfly/react-core": "^1.36.0",
     "leaflet": "^1.3.4",
-    "prop-types": "^15.6.2",
-    "react": "^16.6.0",
-    "react-dom": "^16.6.0",
-    "react-leaflet": "^2.1.1",
-    "react-redux": "^5.1.0",
-    "react-scripts": "2.1.1",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-leaflet": "^2.1.2",
+    "react-redux": "^5.1.1",
+    "react-scripts-ts": "^4.0.8",
     "redux": "^4.0.1",
     "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0",
     "sockjs-client": "^1.3.0",
     "tachyons": "^4.11.1",
-    "webstomp-client": "^1.2.5"
+    "webstomp-client": "^1.2.6"
   },
   "scripts": {
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "lint": "./node_modules/eslint/bin/eslint.js ./src",
-    "lint:fix": "npm run lint -- --fix",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "start": "react-scripts-ts start",
+    "build": "react-scripts-ts build",
+    "test": "react-scripts-ts test --env=jsdom",
+    "lint": "./node_modules/tslint/bin/tslint -c tslint.json 'src/**/*.{ts,tsx}' -e 'src/@types/**'",
+    "eject": "react-scripts-ts eject"
```

Comment:
@danielefiungo please keep `lint:fix` script in place. There is no reason to remove it, it works with TSLint too.

Reply:
done